modules/core/src/v2/coins/trx.ts

```diff
@@ -0,0 +1,54 @@
+const tronweb = require('tronweb');
+
+import { BaseCoin } from '../baseCoin';
+import { BitGo } from '../../bitgo';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+export class Trx extends BaseCoin {
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+
+  constructor(bitgo: BitGo, staticsCoin: Readonly<StaticsBaseCoin>) {
+    super(bitgo);
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  getChain() {
+    return this._staticsCoin.name;
+  }
+
+  getFamily() {
+    return this._staticsCoin.family;
+  }
+
+  getFullName() {
+    return this._staticsCoin.fullName;
+  }
+
+  getBaseFactor() {
+    return new Number('1e' + this._staticsCoin.decimalPlaces);
```

Comment:
`Math.pow(10, this._staticsCoin.decimalPlaces)`

Reply:
eh sure